app/views/gradebooks/speed_grader.html.erb

```diff
@@ -198,9 +200,11 @@
               <%= render partial: "grading_box_extended",
                          locals: { assignment: @assignment } %>
               <span class="turnitin_score_container"></span>
+              <span class="vericite_score_container"></span>
               <a href="#" id="submit_same_score"><br><%= t('links.submit_same_score', "Use this same grade for the resubmission") %></a>
               <a style="display: none;" title="POST" href="<%= context_url(@context, :update_submission_context_gradebook_url) %>" class="update_submission_grade_url">&nbsp;</a>
               <div class="turnitin_info_container"></div>
+              <div class="vericite_info_container"></div>
```

Comment:
Since turnitin and vericite are mutually exclusive, let's use the same container for both.